webapi/src/main/scala/org/knora/webapi/SharedTestDataADM.scala

```diff
@@ -1870,6 +1872,412 @@ object SharedTestDataADM {
             |  }
             |}""".stripMargin
     }
+    def createOntology(projectIri: IRI, label: String): String = {
```

Comment:
Style: Please put a blank line between two methods (here and elsewhere in the file).